src/sagemaker/jumpstart/types.py

```diff
@@ -346,6 +346,35 @@ def to_json(self) -> Dict[str, Any]:
         json_obj = {att: getattr(self, att) for att in self.__slots__ if hasattr(self, att)}
         return json_obj
 
+    def get_instance_specific_environment_variables(self, instance_type: str) -> Dict[str, str]:
+        """Returns instance specific environment variables.
+
+        Not all models and images have instance specific environment variables.
```

Comment:
`Returns empty dict if a model, instance type tuple does not have specific environment variables.`